kernel_gateway/tests/test_gatewayapp.py

```diff
@@ -708,3 +708,74 @@ def test_api_unsupported_method(self):
             raise_error=False
         )
         self.assertEqual(response.code, 404, 'Endpoint which should not exist did not return 404 status code.')
+
+
+class TestConcurrentAPIGatewayApp(TestGatewayAppBase):
+    def setup_app(self):
+        self.app.prespawn_count = 3
+        self.app.api = 'notebook-http'
+        self.app.seed_uri = os.path.join(RESOURCES,
+                                         'kernel_api{}.ipynb'.format(sys.version_info.major))
+
+    @gen_test
+    def test_should_cycle_through_kernels(self):
+        '''Requests should cycle through kernels
```

Comment:
If you one-line the docstrings on tests, they will appear in failures or in verbose output. Multilines do not.